libs/npm/core/src/hooks/api-editor/constants/Settings.ts

```diff
@@ -8,6 +8,6 @@ export const Settings = {
   TopStoryAlert: 'TopStoryAlertId',
   MorningReport: 'MorningReportId',
   FrontPageImagesReport: 'FrontPageImagesReportId',
-  NewsRadioProductFilter: 'NewsRadioProductFilterId',
+  NewsRadioMediaTypeFilter: 'NewsRadioMediaTypeFilterId',
```

Comment:
Make sure you manually update DEV and TEST for this

Reply:
I'll ask you how when you merge the PR ;)